tests/gradients/test_parameter_shift.py

```diff
@@ -1479,12 +1479,13 @@ def test_gradients_agree_finite_differences(self, tol):
         grad_F2 = grad_fn(
             tape, dev, fn=qml.gradients.finite_diff, approx_order=2, strategy="center"
         )
-        grad_A = grad_fn(tape, dev)
+        grad_A = grad_fn(tape, dev, broadcast=True)

```

Comment:
Why was broadcasting changed here?

Reply:
The test is in the broadcasting test class and unless I missed something, it's identical to the test for the vanilla case. So it seems like `broadcast=True` was supposed to be passed originally too. Tagging @dwierichs for awareness too (perhaps this was intentional).

Reply:
Hi @antalszava, this is a great catch, thank you! 
I can not remember a particular reason why `broadcast=True` should not be here, and I agree with you reasoning for why it should be here, so I think this is a typo. :)